README.md

```diff
@@ -73,3 +73,62 @@ Run the command below to clone the projects repository, this will clone both the
    ```
    cd client && npm run dev
    ```
+
```

Comment:
@AbegaM - in the previous section (about `Soul`) we keep on `cd-ing` into `soui/core`, when we're already in it.  Could you fix that?  Aaah, we `nano .env` twice, two. We should just make two changes in one edit session.